docs/guides/endgame.md

```diff
@@ -0,0 +1,199 @@
+# Endgame
+
+## About
+
+Inspired by the release process of [VSCode](https://github.com/Microsoft/vscode/wiki/Development-Process#end-game), Carbon follows a release process that we refer to as the endgame. Given that our work is structured in a series of projects across an iteration, the endgame is an opportunity in the final week of an iteration for us to coordinate final deliverables, run verification testing, and lean on the Carbon community to help verify pre-release builds.
+
+### Meet the endgame team
+
+Every release cycle, we have three team members participate in running the endgame in addition to their project-specific responsibilities. Running the endgame involves a variety of tasks outlined in our [timeline](#timeline). This endgame team is composed of a release leader, and two side-kicks. There should be at minimum one designer and one developer in each endgame team.
```

Comment:
Shouldn't an a11y person be required on the team too if this is the time for regression testing and currently most AVT steps rely on manually tests?

Reply:
@elizabethsjudd this part of the outline talks about the team who is responsible for coordinating various parts of the endgame process, not necessarily the ones who are doing the verification 👍 Definitely agreed representatives should be involved when running the endgame at the end of the iteration cycle.